image/xiaomi_depthmap.ksy

```diff
@@ -0,0 +1,65 @@
+meta:
+  id: xiaomi_depthmap
+  endian: le
+seq:
+  - id: fixed_header
+    type: magic_constant
+  - id: image_info
+    type: info
+  - id: padding
+    type: padding80  
+  - id: depthmap_info
+    type: type_depthmap_info
+  - id: confidence_map
+    type: sector
+    repeat: until
+    repeat-until: not _.has_next
+  - id: depthmap
+    size-eos: true
+    
+types:
+  magic_constant:
+    seq:
+      - id: magic
+        contents: "PMPD"
+      - id: magic2
+        contents: [0x02, 0x00, 0x64, 0x00, 0x01, 0x00, 0x00]
```

Comment:
Is it really a magic? Looks like 3 `u2` and then `u1`

Reply:
Reasonable. I can't be 100% sure as I have that same string on every file I tried. I'll change it.